tortoise/aggregation.py

```diff
@@ -8,14 +8,15 @@
 from pypika.functions import Sum as PypikaSum
 from pypika.functions import Trim as PypikaTrim
 from pypika.terms import AggregateFunction
+from pypika.terms import Function as BaseFunction
 
 from tortoise.exceptions import ConfigurationError
 
 
-class Aggregate:
+class Function:
```

Comment:
:+1:

Reply:
I'm thinking we move this to functions.py, and if you import via aggregate.py we do a `DeprecationWarning` ?

Reply:
Yes, I can do that.

Reply:
done.